src/cli/README.md

```diff
@@ -546,6 +546,176 @@ channel monitor stop
 Done
 ```
 
+### channel manager
```

Comment:
Was including these changes intentional?

Reply:
Should be fixed now. This branch was based off the branch I used for the earlier pull request #6385 adding the CLI documentation. It must not have merged well.